pandas/io/parsers/readers.py

```diff
@@ -382,6 +402,7 @@
     "memory_map": False,
     "error_bad_lines": True,
     "warn_bad_lines": True,
+    "on_bad_lines": None,
```

Comment:
yeah should remove error/warn_bad_lines from here

Reply:
_get_options_with_defaults is really spaghetti-fied right now, so removing this would not make it appear as an option. I will try to clean up _get_options_with_defaults in a future PR if I have time.